src/style.css

```diff
@@ -0,0 +1,80 @@
+html {
+  margin: 0;
+  box-sizing: border-box;
+  font-style: normal;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+@font-face {
+  font-family: Avenir Book;
+  src: url("fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir Heavy;
+  src: url("fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: 400;
```

Comment:
It is better to use the same items of measurement for the whole document (for `font-weight`) be consistent.